timestamp,endpoint,parameters,error,status,misbehavior_detected,misbehavior_reason
2025-07-30T18:41:27.918292+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bef0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:28.420656+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80320>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:28.924018+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81370>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:29.426701+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82e10>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:29.928609+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80800>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:30.430810+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7b980>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:30.932253+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0757cf9d60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:31.465868+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81c10>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:31.967805+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82ab0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:32.469522+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d812b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:32.970742+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82db0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:33.472506+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756cc6750>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:33.973603+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7be60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:34.474920+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d820f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:34.976049+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82cf0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:35.477644+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81c40>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:35.979573+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81520>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:36.481722+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bda0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:36.983480+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756e3e690>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:37.485431+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:37.986947+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d826f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:38.488101+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d832f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:38.989207+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80aa0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:39.491264+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d29a30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:39.993086+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7b9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:40.495157+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80b30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:40.997050+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81400>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:41.498669+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80110>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:42.000188+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83dd0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:42.502343+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bcb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:43.003603+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756ed41d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:43.504935+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83a70>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:44.009189+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80e60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:45.633070+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d808f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:46.134190+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80d40>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:46.635933+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756c9be30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:47.137553+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7be60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:47.639559+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d820f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:48.141757+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82450>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:48.643715+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80740>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:49.145835+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81190>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:49.648122+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bd10>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:50.149889+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0757cf9d60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:50.651775+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82ff0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:51.153435+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82cc0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:51.655187+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81400>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:52.157073+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81940>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:52.659281+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756c9be30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:53.161281+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bdd0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:53.663215+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d801d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:54.165128+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81100>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:54.666581+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80e60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:55.168091+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d837a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:55.669710+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7aae0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:56.171150+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0757b17500>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:56.672413+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81eb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:57.174229+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82c90>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:57.676380+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82450>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:58.178490+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d827b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:58.680909+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756e3c530>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:59.182593+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bfe0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:41:59.684447+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83410>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:00.185683+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d821b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:00.687462+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82b70>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:01.188759+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d821e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:01.690575+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bc20>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:02.191937+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d30d40>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:02.693794+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83440>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:03.195890+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81d90>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:03.697859+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83ef0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:04.199902+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83050>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:04.702974+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d29a30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:05.205380+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7ae40>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:05.707003+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d800b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:06.209064+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d826f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:06.710654+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81790>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:07.212269+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80b30>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:07.714562+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7aae0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:08.216092+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f07589a2330>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:08.717625+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d806b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:09.219299+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d80d70>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:09.721013+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d81520>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:10.222890+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d818e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:10.724361+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0757cf9d60>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:11.225621+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7b980>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:11.726971+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d82000>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:12.228067+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d83890>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:12.729762+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d815b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:13.231412+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d802f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
2025-07-30T18:42:13.733463+00:00,http://localhost:8000/scrape-jobs/,"{'url': 'http://malicious.example.com/payload.py', 'country': 'hacker world'}","HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /scrape-jobs/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0756d7bfb0>: Failed to establish a new connection: [Errno 111] Connection refused'))",error,false,
